### ISSUE TITLE
Make.py fails on Python 3.7

### ISSUE THREAD

**Issue by Teteros** [kunquat/kunquat#810]
$ ./make.py
```py
Traceback (most recent call last):
  File "./make.py", line 26, in <module>
    import support.fabricate as fabricate
  File "/home/tete/code/aurpending/kunquat/support/fabricate.py", line 787
    def __init__(self, async, command):
                           ^
SyntaxError: invalid syntax
```
[What’s New In Python 3.7](https://docs.python.org/3/whatsnew/3.7.html)

> Backwards incompatible syntax changes:
> - async and await are now reserved keywords.

**Comment by kagu** [comment]
Thanks for letting me know! See if the associated pull request fixes the problem for you.

**Comment by Teteros** [comment]
The pull request branch compiles, thank you for the quick fix!
Strangely on Arch, and I haven't encountered the newer gcc issue mentioned in #801.

(If you use `fixes #810` or another [keyword](https://help.github.com/articles/closing-issues-using-keywords) on the PR description it wi;; close this issue automatically after merging)

**Comment by kagu** [comment]
Thanks for the tip on keywords :-) The GCC issues mentioned in #801 were fixed by the pull request #800.
